src/GitElephant/Command/BaseCommand.php

```diff
@@ -293,7 +293,6 @@ protected function addPath($path): void
     public function normalizeOptions(array $options = array(), array $switchOptions = array(), $valueOptions = array()): array
     {
         $normalizedOptions = array();
-
```

Comment:
What is the motivation for removing these spaces?

I see two downsides here:
1. it's harder to focus on code changes related to bugfix
2. these spaces separate code blocks by 'logic', removing spaces will decrease code readability, in my experience.. but may be im just old and picky )

Reply:
Like here https://www.php-fig.org/psr/psr-12/ see? separate code blocks by "logical blocks" ;)

Reply:
I admit that this is mostly the cs-fixer rather than myself manually. I think I get your point though and will look for a cs fixer rule to enforce this also in the long run.